main/autogen/data/msgpack.cc

```diff
@@ -170,26 +170,35 @@ string MsgpackWriter::pack(core::Context ctx, ParsedFile &pf) {
 
     packString("symbols");
     int i = -1;
+    int numTypes = typeCount.at(version);
     mpack_start_array(&writer, symbols.size());
     for (auto sym : symbols) {
         ++i;
         string str;
-        switch ((Definition::Type)i) {
-            case Definition::Type::Module:
-                str = "module";
-                break;
-            case Definition::Type::Class:
-                str = "class";
-                break;
-            case Definition::Type::Casgn:
-                str = "casgn";
-                break;
-            case Definition::Type::Alias:
-                str = "alias";
-                break;
-            default:
-                str = sym.show(ctx);
+        if (i < numTypes) {
+            switch ((Definition::Type)i) {
```

Comment:
Depending on the version, we serialize a different number of definition types

Reply:
As long as we're here, would you mind changing this to a C++-style cast?

```suggestion
            switch (static_cast<Definition::Type>(i)) {
```

Reply:
Ah sorry just saw, can fix this later